src/components/editor/InputTarget.jsx

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react';
+import {Typeahead} from 'react-bootstrap-typeahead'
+import PropTypes from 'prop-types'
+
+class InputTarget extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isLoading: true,
+      options: [],
+      selected: []
+    }
+  }
+
+  render() {
+    const target_url = './static/spoofedFilesFromServer/fromQA/frequencies.json'
+
+    return (
+      <div>
+        <label htmlFor="targetComponent">{this.props.propertyTemplate.propertyLabel}
+        <Typeahead
+          id="targetComponent"
+          required={JSON.parse(this.props.propertyTemplate.mandatory)}
+          placeholder={this.props.propertyTemplate.propertyLabel}
+          multiple={JSON.parse(this.props.propertyTemplate.repeatable)}
+          useCache={true}
```

Comment:
same "ew" on having to put `true` in {}.  Do we really have to?  (this is not only spot, so I'm guessing no?)